examples/cactus-example-cbdc-bridging-backend/cache/solidity-files-cache.json

```diff
@@ -0,0 +1 @@
terface":{"0.8.20":{"path":"satp-contract-interface.sol/SATPContractInterface.json","build_id":"533d2c5e3c1a73b85c754b76cd8b9ca2"}}},"seenByCompiler":true},"src/solidity/main/satp-erc20.sol":{"lastModificationDate":1725357801969,"contentHash":"5ecdb78db9a7b063ee9e0a5702733bd6","sourceName":"src/solidity/main/satp-erc20.sol","compilerSettings":{"solc":{"optimizer":{"enabled":true,"runs":200},"metadata":{"useLiteralContent":false,"bytecodeHash":"ipfs","appendCBOR":true},"outputSelection":{"*":{"*":["abi","evm.bytecode","evm.deployedBytecode","evm.methodIdentifiers","metadata"]}},"evmVersion":"paris","viaIR":false,"libraries":{}},"vyper":{"evmVersion":"paris","outputSelection":{"*":{"*":["abi","evm.bytecode","evm.deployedBytecode"]}}}},"imports":["node_modules/@openzeppelin/contracts/access/AccessControl.sol","node_modules/@openzeppelin/contracts/access/IAccessControl.sol","node_modules/@openzeppelin/contracts/interfaces/draft-IERC6093.sol","node_modules/@openzeppelin/contracts/token/ERC20/ERC20.sol","node_modules/@openzeppelin/contracts/token/ERC20/IERC20.sol","node_modules/@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol","node_modules/@openzeppelin/contracts/utils/Context.sol","node_modules/@openzeppelin/contracts/utils/Strings.sol","node_modules/@openzeppelin/contracts/utils/introspection/ERC165.sol","node_modules/@openzeppelin/contracts/utils/introspection/IERC165.sol","node_modules/@openzeppelin/contracts/utils/math/Math.sol","node_modules/@openzeppelin/contracts/utils/math/SignedMath.sol","src/solidity/main/ITraceableContract.sol","src/solidity/main/satp-contract-interface.sol"],"versionRequirement":"^0.8.20","artifacts":{"SATPContract":{"0.8.20":{"path":"satp-erc20.sol/SATPContract.json","build_id":"533d2c5e3c1a73b85c754b76cd8b9ca2"}}},"seenByCompiler":true},"src/solidity/main/satp-wrapper.sol":{"lastModificationDate":1725357617051,"contentHash":"3796cb4be7c06c5c8912e8137293b8a9","sourceName":"src/solidity/main/satp-wrapper.sol","compilerSettings":{"solc":{"optimizer":{"enabled":true,"runs":200},"metadata":{"useLiteralContent":false,"bytecodeHash":"ipfs","appendCBOR":true},"outputSelection":{"*":{"*":["abi","evm.bytecode","evm.deployedBytecode","evm.methodIdentifiers","metadata"]}},"evmVersion":"paris","viaIR":false,"libraries":{}},"vyper":{"evmVersion":"paris","outputSelection":{"*":{"*":["abi","evm.bytecode","evm.deployedBytecode"]}}}},"imports":["node_modules/@openzeppelin/contracts/access/Ownable.sol","node_modules/@openzeppelin/contracts/utils/Context.sol","node_modules/@openzeppelin/contracts/utils/Strings.sol","node_modules/@openzeppelin/contracts/utils/math/Math.sol","node_modules/@openzeppelin/contracts/utils/math/SignedMath.sol","src/solidity/main/ITraceableContract.sol"],"versionRequirement":"^0.8.20","artifacts":{"SATPWrapperContract":{"0.8.20":{"path":"satp-wrapper.sol/SATPWrapperContract.json","build_id":"533d2c5e3c1a73b85c754b76cd8b9ca2"}}},"seenByCompiler":true}},"builds":["533d2c5e3c1a73b85c754b76cd8b9ca2"]}
```

Comment:
@LordKubaya I recommend ignoring build cache artifacts like this in the gitignore file

Reply:
Done, thank you!